<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AchieveWindow</class>
 <widget class="QMainWindow" name="AchieveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成就</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rec.qrc">
    <normaloff>:/myimages/images/achievementicon.png</normaloff>:/myimages/images/achievementicon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="delete_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>500</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>点击此处清空数据-&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>1200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>1200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>630</y>
      <width>1200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="level1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LEVEL 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="level2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LEVEL 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="level3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LEVEL 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="level4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LEVEL 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lock">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rec.qrc">:/myimages/images/lock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unlock">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>49</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rec.qrc">:/myimages/images/unlock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lockLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>151</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>未通关</string>
    </property>
   </widget>
   <widget class="QLabel" name="unlockLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>45</y>
      <width>81</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>通关</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>706</x>
      <y>0</y>
      <width>16</width>
      <height>90</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="delete_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>351</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>创造你的最佳记录！</string>
    </property>
   </widget>
   <widget class="QLabel" name="time1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>135</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>315</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>495</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>675</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>135</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>步数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>315</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>步数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>495</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>步数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>675</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>步数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time3_">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>131</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
